01-filedir.md

```diff
@@ -348,6 +348,45 @@ it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
 but we'll see some uses for it soon.
+
+`cd` also allows you switch back and forth between places in a rapid way,
+using `cd -`.
+Much like `..`, `-` is a special instruction that `cd` translates to
+*the previous directory I was in*.
+For example, if we are currently in Nelle's `data` directory,
+
+~~~ {.bash}
+$ pwd
+~~~
+~~~ {.output}
+/Users/nelle/data
+~~~
+
+we can move to Nell's Desktop with
+
+~~~ {.bash}
+$ cd /Users/nelle/Desktop
+$ pwd
+~~~
+~~~ {.output}
+/Users/nelle/Desktop
+~~~
+
+If we want to go back to the `data` directory,
+we just need to type `cd -`:
+
+~~~ {.bash}
+$ cd -
+$ pwd
+~~~
+~~~ {.output}
+/Users/nelle/data
+~~~
+
+This is a *very* efficient way of moving back and forth between directories. 
+The difference between `cd ..` and `cd -` is that the former brings you *up*,
```

Comment:
"the previous directory I was in" at line 355 is very explanatory to me so I don't see the need to explain it again. But I'm OK leaving this as is.